# HG changeset patch
# Parent 7c4c24a8af98cdf95ca596cc6599efb0678b20c0
# User Frédéric Wang <fred.wang@free.fr>
Change comments about TeX/Mathematica fonts to STIX/Asana fonts

diff --git a/layout/mathml/mathml.css b/layout/mathml/mathml.css
--- a/layout/mathml/mathml.css
+++ b/layout/mathml/mathml.css
@@ -380,26 +380,26 @@ mtd[_moz-math-columnline="dashed"] {
 /* Style used for stretchy symbols *must* be normal to avoid misaligments */
 /* By leaving the font-family empty, the MathML engine will use the value
    provided by the mathfont-family property in the mathfont.properties file
    or the value of the user's pref("font.mathfont-family", "...").
 
    Authors can make elements on a document to be stretched with different
    fonts, e.g.,
 
-   To request the use of TeX fonts, you can add a <style>...</style> with:
-   <mo myfonts="tex">...</mo> with the associated CSS declaration
-   mo[myfonts="tex"]::-moz-math-stretchy {
-     font-family: CMSY10, CMEX10;
+   To request the use of STIX fonts, you can add a <style>...</style> with:
+   <mo myfonts="stix">...</mo> with the associated CSS declaration
+   mo[myfonts="stix"]::-moz-math-stretchy {
+     font-family: STIXNonUnicode, STIXSizeOneSym, STIXSize1, STIXGeneral;
    }
 
-   To request the use of Mathematica fonts, you can add a <style>...</style> with:
-   <mo myfonts="mathematica">...</mo> with the associated CSS declaration
-   mo[myfonts="mathematica"]::-moz-math-stretchy {
-     font-family: Math1, Math2, Math4;
+   To request the use of Asana fonts, you can add a <style>...</style> with:
+   <mo myfonts="asana">...</mo> with the associated CSS declaration
+   mo[myfonts="asana"]::-moz-math-stretchy {
+     font-family: Asana Math;
    }
 
    Of course, if you just want all of the stretchy characters in your
    document to be stretched with your preferred list, you can just do:
    ::-moz-math-stretchy {
      font-family: [your-particular-list]
    }
 
